src/wordpress/generator.py

```diff
@@ -205,10 +205,10 @@ def generate(self, deactivated_plugins=None):
         theme = WPThemeConfig(self.wp_site, settings.DEFAULT_THEME_NAME, self._site_params['theme_faculty'])
         theme.install_and_activate()
 
-        logging.info("%s - Activating theme '%s'...", repr(self), self._site_params['theme'])
+        logging.info("%s - Activating theme '%s'...", repr(self), settings.DEFAULT_THEME_NAME)
```

Comment:
Pourquoi t'as changé cette partie ? parce que là, à priori, peu importe ce que tu affiches là, ce qui fiat fait foi, c'est ce qui est activé par `theme.activate()` et qui n'est pas forcément le DEFAULT_THEME_NAME...

Reply:
Je ne me souviens pas avoir changé cette partie ... github "in the fraises" ... je vais annulé les modifications